src/content/blog/2024_zne_catalyst.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Just in time for just-in-time error mitigation"
```

Comment:
Minor comment, but as the title is quite important for SEO, could be good to give it a 'subtitle'. E.g.,

> Just in time for just-in-time error mitigation: compiling error mitigation with LLVM, X, and Y.

Could sprinkle in important keywords as well like Mitiq, Catalyst, etc.